contracts/scripts/lifebank.sh

```diff
@@ -51,17 +51,17 @@ create_lifebank_accounts() {
         cleos -u $EOS_API_URL system newaccount eosio \
             --transfer $account \
             $EOS_PUB_KEY \
-            --stake-net "10000.0000 EOS" \
```

Comment:
@JustinCast `EOS` is the system token, i think `EOS` does have 4 decimals of precision.

the `LIFE` token has zero